core/pom.xml

```diff
@@ -468,6 +468,11 @@ THE SOFTWARE.
       <artifactId>commons-logging</artifactId>
     </dependency>
     <dependency>
+      <groupId>log4j</groupId>
+      <artifactId>log4j</artifactId>
+      <version>1.2.17</version>
```

Comment:
Hmm, most other dependencies don't use a property, so shouldn't this change be consistent with the existing configuration?